plugins/outputs/cloudwatchlogs/internal/pusher/queue_test.go

```diff
@@ -86,6 +86,10 @@ func (m *mockSender) RetryDuration() time.Duration {
 	return args.Get(0).(time.Duration)
 }
 
+func (m *mockSender) Stop() {
+
```

Comment:
Think it'd be worth asserting this is called when the pusher is stopped in one of the tests.